docs/user-guide/how-to-deploy.rst

```diff
@@ -51,460 +51,13 @@ Docker Compose
-image and add your custom dependencies. For example:
-
-.. code-block:: docker
-
-    # Extend the official Rasa SDK image
-    FROM rasa/rasa-sdk:latest
-
-    # Add a custom system library (e.g. git)
-    RUN apt-get update && \
-        apt-get install -y git
-
-    # Add a custom python library (e.g. jupyter)
-    RUN pip install --no-cache-dir jupyter
-
-You can then build the image via the following command, and use it in your
-``docker-compose.yml`` instead of the ``rasa/rasa-sdk`` image.
-
-.. code-block:: bash
-
-  docker build . -t <name of your custom image>:<tag of your custom image>
-
-Adding a Custom Tracker Store
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-By default, all conversations are saved in memory. This means that all
-conversations are lost as soon as you restart the Rasa server.
-If you want to persist your conversations, you can use a different
-:ref:`Tracker Store <tracker-stores>`.
-
-Using PostgreSQL as Tracker Store
-*********************************
-
-Start by adding PostgreSQL to your docker-compose file:
-
-.. code-block:: yaml
-
-  postgres:
-    image: postgres:latest
-
-Then add PostgreSQL to the ``tracker_store`` section of your endpoint
-configuration ``config/endpoints.yml``:
-
-.. code-block:: yaml
-
-  tracker_store:
-    type: sql
-    dialect: "postgresql"
-    url: postgres
-    db: rasa
-
-Using MongoDB as Tracker Store
-******************************
-
-Start by adding MongoDB to your docker-compose file. The following example
-adds the MongoDB as well as a UI (you can skip this), which will be available
-at ``localhost:8081``. Username and password for the MongoDB instance are
-specified as ``rasa`` and ``example``.
-
-.. code-block:: yaml
-
-  mongo:
-    image: mongo
-    environment:
-      MONGO_INITDB_ROOT_USERNAME: rasa
-      MONGO_INITDB_ROOT_PASSWORD: example
-  mongo-express:
-    image: mongo-express
-    ports:
-      - 8081:8081
-    environment:
-      ME_CONFIG_MONGODB_ADMINUSERNAME: rasa
-      ME_CONFIG_MONGODB_ADMINPASSWORD: example
-
-Then add the MongoDB to the ``tracker_store`` section of your endpoints
-configuration ``endpoints.yml``:
-
-.. code-block:: yaml
-
-  tracker_store:
-    type: mongod
-    url: mongodb://mongo:27017
-    username: rasa
-    password: example
-
-Then start all components with ``docker-compose up``.
-
-Using Redis as Tracker Store
-****************************
-
-Start by adding Redis to your docker-compose file:
-
-.. code-block:: yaml
-
-  redis:
-    image: redis:latest
-
-Then add Redis to the ``tracker_store`` section of your endpoint
-configuration ``endpoints.yml``:
-
-.. code-block:: yaml
-
-  tracker_store:
-    type: redis
-    url: redis
-
-Using a Custom Tracker Store Implementation
-*******************************************
-
-If you have a custom implementation of a tracker store you have two options
-to add this store to Rasa:
-
-  - extending the Rasa image
-  - mounting it as volume
+.. toctree::
+   :titlesonly:
+   :maxdepth: 1
```

Comment:
@akelad When it comes to the actual information split of what goes where, it still needs work. But I wanted your feedback on what's left of this page. For the bottom part here, I think we have 2 options, the first would be the text I put with no in-line links but with the TOC list below. The other would be the text, with inline links, with no TOC. Build the docs and let me know what you think?

Also, I think that the "Rasa-Only deployment with docker-compose" still looks too big in this info heirarchy. Since it's the same size as "recommended deployment methods", i think it kinda defeats the purpose of trying to push that it's not a recommended deployment method...

Reply:
1. Would go with no TOC, just the in line comments. 
2. Hm, could we call the section "alternative deployment" ? And then just say you can also do a Rasa Open Source only deployment